### PR TITLE
Add HW16

### PR DESCRIPTION
**Pull request by Gritskov-Vitaliy**

### CODE REVIEW

File: HW16/Dockerfile_Network [hunk 1]
```diff
@@ -0,0 +1,2 @@
+FROM ubuntu
+RUN apt-get update && apt-get install -y iputils-ping
```
Comment (mavrik90) [inline HW16/Dockerfile_Network]: Cleanup stage?

File: HW16/bash_script.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```
Comment (mavrik90) [inline HW16/bash_script.sh:1]: Second way?

File: HW16/script.sh [hunk 3]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+ 
+A=$1
+B=$2
+C=$3
+
+if [ -z $A ] || [ -z $B ] || [ -z $C ] || [[ ! $A == ?(-)+([0-9]) ]] || [[ ! $B == ?(-)+([0-9]) ]] || [[ ! $C == ?(-)+([0-9]) ]]; then
+    echo "The value of A, B, or C is empty or is not a number" 
+elif (( $A == $B )); then
```
Comment (mavrik90) [inline HW16/script.sh]: To make your code smaller create a function to compare two variables. Build your output based on function outputs.

File: HW16/Dockerfile_Network (hunk 1 above)
Comment (Gritskov-Vitaliy) [reply HW16/Dockerfile_Network]: Done.
image
image
image

File: HW16/bash_script.sh (hunk 2 above)
Comment (Gritskov-Vitaliy) [reply HW16/bash_script.sh:1]: Done
image

File: HW16/script.sh (hunk 3 above)
Comment (Gritskov-Vitaliy) [reply HW16/script.sh]: Done
image

**Review by mavrik90 — APPROVED** [review]
LGTM

**Pull request merged**